src/internal/deploy/cmds/cmds.go

```diff
@@ -537,7 +546,19 @@ func standardDeployCmds() []*cobra.Command {
 				// Serve the Pachyderm object/block API locally, as this is needed by
 				// our tests (and authentication is disabled anyway)
 				opts.ExposeObjectAPI = true
```

Comment:
I thought the ObjectAPI was gone?

Reply:
If so we should make an issue to remove all the references to it.

Reply:
Yeah I think that's my bad, I didn't get all the configuration, when I removed it.  Just confirming it's not used for anything here.

Reply:
I threw this onto the 2.0 board. https://github.com/pachyderm/pachyderm/projects/11#card-57128360